ArraysHomework/ArraysHomework.cs

```diff
@@ -72,6 +78,8 @@ public void Arrays()
                 array2[down] = down;
                 
             }
+            int[] combined = new int[50];
+            combined = array1.Concat(array2).ToArray();
```

Comment:
This is a good answer, but a bit ahead of the topic. How would you do this just using the flow control and array access/assignment statements that we covered?